### PR TITLE
edit templates/NOTES.txt

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: templates/NOTES.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@
 {{- if gt (len $movedList) 0 }}
 Deprecated: The Helm values below should be updated to avoid breaking in a future release!
 {{- if hasKey .Values "cemanager" }}
-  ⚠ The "cemanager" Kubernetes Service will be renamed to "coderd" once moved.
+  ⚠ The "cemanager" Kubernetes Service will be renamed to "coderd".
```
Comment (khorne3) [inline templates/NOTES.txt:27]: Not sure if this is accurate but I thought "once moved" was confusing

Comment (kylecarbs) [reply templates/NOTES.txt:27]: This triggers if any `cemanager.*` keys exist in the helm. I wasn't sure how to word it, but the service automatically gets renamed to "coderd" once the old keys have been renamed.

Comment (khorne3) [reply templates/NOTES.txt:27]: Maybe `once all cemanager.* Helm values are renamed, the "cemanager" Kubernetes Service will be renamed to "coderd"`?

**Review by jawnsy — APPROVED** [review]
These are minor text changes, so LGTM.

**Pull request merged**